gdal-src/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "gdal-src"
-version = "0.1.0+3.10.2"
+version = "0.2.0"
```

Comment:
I think we can keep this as `0.2.0+3.10.2`. I forgot that we already had (I think?) an empty version published.

Reply:
the last gdal-src is just 0.1.0 see https://crates.io/crates/gdal-src/versions

Reply:
Yeah, you published it so we could ask for a crate size exemption. We can do `0.2.0+3.10.2`, `0.3.0+3.11.0` and so onn.

Reply:
cargo release keeps complaining about the version but we can try it. Maybe it is just a dry-run issue.

Reply:
Does it warn that the extra part is useless when you try to publish the crate? I think that's right, but harmless.